source/host.c

```diff
@@ -507,6 +513,10 @@ extern float crosshair_offset_step;
 void Host_ClearMemory (void)
 {
 	Con_DPrintf ("Clearing memory\n");
+	
+#ifdef __WII__
+	GL_ClearTextureCache();
```

Comment:
Can you explain this function? Is this something other platforms can benefit from?

Reply:
This platform uses a separate heap for textures which is stored in MEM2 (ideally). This function attempts to clear and flush all texture data stored in that heap, which should mitigate double storage of textures (if it occurs), or extra textures stuck in memory on a new map load.